recipes/libsvm/all/CMakeLists.txt

```
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 2.8)
+project(svm C CXX)
+
+include(../conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_library(svm svm.cpp)
+set_target_properties(svm PROPERTIES PUBLIC_HEADER svm.h)
+
+add_executable(svm-predict svm-predict.c)
+target_link_libraries(svm-predict svm)
+
+add_executable(svm-train svm-train.c)
+target_link_libraries(svm-train svm)
+
+add_executable(svm-scale svm-scale.c)
```

```suggestion
add_library(svm source_subfolder/svm.cpp)
set_target_properties(svm PROPERTIES PUBLIC_HEADER source_subfolder/svm.h)
target_include_directories(svm PRIVATE source_subfolder)

add_executable(svm-predict source_subfolder/svm-predict.c)
target_link_libraries(svm-predict svm)

add_executable(svm-train source_subfolder/svm-train.c)
target_link_libraries(svm-train svm)

add_executable(svm-scale source_subfolder/svm-scale.c)
```

You also can run cmake from build_folder, so you don't need to move it to source_subfolder.